x-pack/apm-server/main_test.go

```diff
@@ -7,7 +7,13 @@ package main
 // This file is mandatory as otherwise the apm-server.test binary is not generated correctly.
 
 import (
+	"bytes"
 	"context"
+	"fmt"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
```

Comment:
Please run `make fmt` (it'll run goimports) to move these down a block

Reply:
The IDE is playing with me: every time it upgrades, my Go settings (which include `goimports` formatting) are erased...
Thanks for letting me now 👍🏼